### DISCUSSION TITLE
LF Support for <dt> and <dd> elements

### DISCUSSION THREAD

**Discussion by dalvarez** [vercel/next.js#33965] [Help]
I'm trying to create a definition block in my code and it seems that next js does not know how to handle the `<dt>` and `<dd>` tags?

code example:
```      
<dfn>
  <dt>{term}</dt>
   <dd>{definition}</dd>
</dfn>
```      

console error:
```next-dev.js?3515:32 Warning: Expected server HTML to contain a matching <dt> in <dfn>```

it seems to be fine with the `<dfn>` element, but not the semantic children

**Comment by wwwdepot** [comment]
The syntax is the following [according to W3C](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element)


```jsx
      <dl>
        <dt>
          <dfn>Apartment</dfn>, n.
        </dt>
        <dd>
          An execution context grouping one or more threads with one or more COM
          objects.
        </dd>

        <dt>
          <dfn>Flat</dfn>, n.
        </dt>
        <dd>A deflated tire.</dd>

        <dt>
          <dfn>Home</dfn>, n.
        </dt>
        <dd>The user's login directory.</dd>
      </dl>
```
